docs/rails.md

```diff
@@ -0,0 +1,35 @@
+# Rails conventions
+
+## App behavior should be configured via environment variables
+
+When implementing new features, it should not depend on current
+`RAILS_ENV` value, but environment variable(s) specific to the
+feature.
+
+* Avoid modifying `config/environments/*.rb`, and keep default
+  rails configuration (this will also help when updating rails);
+* Avoid testing:
+  + `Rails.env.development?`;
+  + `Rails.env.production?`;
+  + `Rails.env.test?`.
+
+Example:
```

Comment:
What about a good and a bad example, like Rubocop documentation does?

Reply:
Done.

I hesitated to include a "bad one", as some people don't like showing "what we
shouldn't do".